Basic-Audio-Transformer/js/transformer.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-unused-vars */
+/* global Float32Array AudioData */
+
+export default class AudioTransformer {
+  lastValuePerChannel = null;
+  cutoff = 100;
+
+  // eslint-disable-next-line class-methods-use-this
+  start() {}
+
+  transform(data, controller) {
+    const format = 'f32-planar';
```

Comment:
If we got this recipe from somewhere, it'll be nice to give props.

Reply:
This was done by Mike in a private PR so not sure how to give him props in this case?

Reply:
Did Mike grab this from somewhere?  Can you ask him please?

Reply:
@maikthomas can you please let us know about this audio transformer filter? Could we just credit your GitHub account?

Reply:
Maybe you can ask Mike offline?

Reply:
https://github.com/webrtc/samples/blob/gh-pages/src/content/insertable-streams/audio-processing/js/main.js

Reply:
Thanks Mike, I added the link in the comments